velox/exec/tests/utils/PlanBuilder.cpp

```diff
@@ -135,6 +135,8 @@ PlanBuilder& PlanBuilder::tpchTableScan(
 }
 
 core::PlanNodePtr PlanBuilder::TableScanBuilder::build(core::PlanNodeId id) {
+  VELOX_CHECK_NOT_NULL(
+      outputType_, "outputType must be set before building TableScanNode");
```

Comment:
`outputType must be specified`
We don't have to specify `before building TableScanNode` because the error will have the call context pointing to `TableScanBuilder::build`

Reply:
removed the superfluous context